Chandra/cmd_states/cmd_states.py

```diff
@@ -245,7 +245,13 @@ def get_states(state0, cmds, exclude=None):
 
         # ACIS power command section
         elif cmd_type == 'ACISPKT':
-            if tlmsid.startswith('WSPOW'):
+            # The power-down command is a special-case that also turns off the vid board
+            if tlmsid == 'WSPOW00000':
```

Comment:
Would it be desirable to fix this within `decode_power`?  At least I usually think of that routine as being generally applicable and not just a private function of this module.

Reply:
I agree with @taldcroft here.

Reply:
Sure.   Though if we consider it a public function I suppose I have to look around and see if anything is using it before changing API / adding vid board status to it.

Reply:
Yes, good plan to check.  It seems somewhat unlikely that anything could be broken by adding a key/value to the output dict, but who knows.

Reply:
OK, I can't find any external use in our code.  I also pinged @richardjedgar because I recalled pointing him to this routine to call from some external testing script and he had no problems with adding keys to the output dict as you say.